docs/BR.md

```diff
@@ -2027,10 +2032,10 @@ The Audit Report MUST contain at least the following clearly-labelled informatio
 4. audit criteria, with version number(s), that were used to audit each of the certificates (and associated keys);
 5. a list of the CA policy documents, with version numbers, referenced during the audit;
 6. whether the audit assessed a period of time or a point in time;
-7. the start date and end date of the period, for those that cover a period of time;
+7. the start date and end date of the period that was audited, for those that cover a period of time (please note, this is not the period that the auditor was on site);
```

Comment:
```suggestion
7. the start date and end date of the Audit Period, for those that cover a period of time;
```

"Audit Period" is defined in Section 1.6.1 as
>  In a period-of-time audit, the period between the first day (start) and the last day of
operations (end) covered by the auditors in their engagement. (This is not the same as the period of time
when the auditors are on-site at the CA.) The coverage rules and maximum length of audit periods are defined
in section 8.1.

This was defined in https://cabforum.org/2017/04/17/ballot-196-define-audit-period/